app/app.go

```diff
@@ -181,7 +183,14 @@ func NewWasmApp(
 	// better way to get this dir???
 	homeDir := viper.GetString(cli.HomeFlag)
 	wasmDir := filepath.Join(homeDir, "wasm")
-	app.wasmKeeper = wasm.NewKeeper(app.cdc, keys[wasm.StoreKey], app.accountKeeper, app.bankKeeper, wasmRouter, wasmDir)
+
+	// Read LRU cache size from config (app.toml), defaults to `0`
```

Comment:
app.toml is parsed into a Config object here: https://github.com/cosmos/cosmos-sdk/blob/4d0dfe592b45b658d796d9b2eb6f2eb5cfe68fc1/server/config/config.go#L15-L45
We don't need to update that as well somehow?

Reply:
Also, I note these have no defaults...
EIther we need defaults here, or (probably better) on the `WasmConfig` struct.

I would propose defaults: 

```go
WasmConfig {
  SmartQueryGasLimit: 300000,
  CacheSize: 0,
}
```

LRU cache should be opt-in. It should be impossible to set 0 gas limit (1 okay) for smart queries, so this means no issue with "unable to set 0 in config" issues.